### PR TITLE
Enable PyTorch UNet CPU benchmark

### PR DESCRIPTION
**Pull request by mariecwhite**
Now that https://github.com/openxla/iree/issues/11447 is fixed, we can enable UNet CPU benchmarks.

benchmarks: x86_64, cuda

### CODE REVIEW

File: tests/e2e/test_artifacts/generated_e2e_test_iree_artifacts.cmake
Comment (pzread) [inline tests/e2e/test_artifacts/generated_e2e_test_iree_artifacts.cmake:1]: We did some changes on how the cmake files are generated. Please regenerate the files before merging

File: build_tools/python/e2e_test_framework/models/model_groups.py
Comment (pzread) [inline build_tools/python/e2e_test_framework/models/model_groups.py:34]: The x86 benchmarks now take 46 mins to run. Seems fine to me but we probably need to fix this before adding more models (either using more machines to run benchmarks or trying to cut some experimental benchmarks)

**Review by pzread — CHANGES_REQUESTED** [review]

File: tests/e2e/test_artifacts/generated_e2e_test_iree_artifacts.cmake
Comment (mariecwhite) [reply tests/e2e/test_artifacts/generated_e2e_test_iree_artifacts.cmake:1]: Updated

File: build_tools/python/e2e_test_framework/models/model_groups.py
Comment (mariecwhite) [reply build_tools/python/e2e_test_framework/models/model_groups.py:34]: Ok I'll hold off on adding more x86 benchmarks.

**Review by pzread — APPROVED** [review]

**Pull request merged**
